CITi-UFPE/CITiMarco
# Hotfix/delete-inputs-page

## What I've done

- removed inputs page component and fixed props validation of table inputs component

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. git checkout develop
   1. git pull
   1. git checkout <branch name>
   1. git pull origin <branch name>
   1. yarn install
   1. code .
   1. yarn start

1. In your browser go to http://localhost:3000/

1. Check if everything is working

Resolves: #23
